README.md

```diff
@@ -102,7 +102,9 @@ theme by providing a key that matches the regex `\w+_variables` (e.g. `alt_varia
 stylesheets will then be generated using the variables at that key.
 
 To use these stylesheets in a template, pass the theme name to the `{% bulma %}` tag either as a
-string `{% bulma 'alt' %}` or as a template variable `{% bulma theme %}`.
+string `{% bulma 'alt' %}` or as a template variable `{% bulma theme %}`. When calling the `bulma` template
+more than once in the same document, you may want to pass `include_js=False` as well to prevent duplicate
+loading of JavaScript resources.
```

Comment:
```suggestion
string `{% bulma 'alt' %}` or as a template variable `{% bulma theme %}`. When calling the `bulma` template
more than once in the same document (for example for implementing a dark theme switch), you will want to pass `include_js=False` to 
at least one of these to prevent duplicate loading of JavaScript resources.
```
Giving an example here seems to make it more clear what the point of this is.